specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2021-11-20-preview/examples/ElasticSans_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,98 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaaaaaaa",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "0Dk_bd2172_UFvY6Y",
+    "parameters": {
+      "properties": {
+        "sku": {
+          "name": "Standard_LRS",
+          "tier": "Standard"
+        },
+        "availabilityZones": [
+          "aaaaaaaaa"
+        ],
+        "baseSizeTB": 12,
+        "extendedSizeTB": 6
+      },
+      "location": "aaaaa",
+      "tags": {
+        "key3755": "aaaaaaaaa"
+      }
+    },
+    "api-version": "2021-11-20-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "sku": {
+            "name": "Standard_LRS",
+            "tier": "Standard"
+          },
+          "availabilityZones": [
+            "aaaaaaaaa"
+          ],
+          "provisioningState": "Invalid",
+          "baseSizeTB": 12,
+          "extendedSizeTB": 6,
+          "totalSizeTB": 7,
+          "provisionedIops": 26,
+          "provisionedMBps": 10
+        },
+        "systemData": {
+          "createdBy": "aaaaaaaaaaaaaaaa",
+          "createdByType": "User",
+          "createdAt": "2022-03-09T06:06:58.232Z",
+          "lastModifiedBy": "aaaaaaaaaaaaaa",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2022-03-09T06:06:58.232Z"
+        },
+        "location": "aaaaa",
+        "id": "aaaaaaaaaaa",
+        "name": "aaaaaaaa",
```

Comment:
>"name" [](http://example.com/codeflow?start=8&length=6)

this name should match the request parameter

Reply:
Make sure to update this on all your examples.

Reply:
Updated.